packages/connect-explorer/src/pages/methods/other/init.mdx

```diff
@@ -32,15 +41,28 @@ export const paramDescriptions = {
         'default `false`. Postpone iframe creation until TrezorConnect is called for the first time.',
     popup: 'default `true`. Projects running on trusted domains (trezor.io) are not required to use popup. For other domains this option is ignored.',
     transports:
-        'Array of transports that should be use. If not provided, TrezorConnect will chose a reasonable default based on your environment.',
+        'Array of transports that should be use. If not provided, TrezorConnect will choose a reasonable default based on your environment.',
+    coreMode:
+        'default `auto`. In `iframe` mode, the Connect core is loaded inside an iframe injected into the page. If set to `popup`, TrezorConnect will avoid using the iframe, instead loading the core in popup. The advantage is that popup enables WebUSB to work cross-origin on the web. The disadvantage is that popup does not retain a stateful connection to the device, which can be a problem for some use-cases. If set to `auto` TrezorConnect will automatically use `iframe` if bridge is available, `popup` otherwise.',
```

Comment:
@mroz22 @Hannsek new text

Reply:
> which can be a problem for some use-cases

Which problems exactly? 🤔

Reply:
For example when you want to receive events about the device

Reply:
I tried suggested some text here:
https://github.com/trezor/trezor-suite/pull/13292#discussion_r1677783635

Reply:
I'd specify these things that won't be available when using `popup` mode.

Reply:
Ok I will update it based on your suggestion

Reply:
But please add there some warning that _some_ things won't be available so it is crystal clear. :)